en/docs/get-started/architecture.md

```diff
@@ -401,7 +401,7 @@ above figure and video.
 <p>WSO2 Identity Server provides a friendly user interface for policy editing. It also supports multiple Policy Information Point (PIP) and policy distribution to various Policy Decision Points (PDPs). It provides a high-performance network protocol (over Thrift) for PEP/PDP interaction, and policy decision and attribute caching. Notifications are provided for policy updates. Moreover, the WSO2 Carbon TryIt tool that comes bundled with the Identity Server lets the user explore the policy impact.</p>
 <div class="admonition info">
 <p class="admonition-title">Related Links</p>
-<p>For more information on how to use and manage entitlement within the Identity Server, see <a href="../../learn/working-with-entitlement">Working with Entitlement</a> .</p>
+<p>For more information on how to use and manage entitlement within the Identity Server, see <a href="../../learn/access-control">Working with Entitlement</a> .</p>
```

Comment:
I think this file has been renamed to access-control-overview now. Please check.

Reply:
the filename of the page has not been changed..only the navigation title